sw/airborne/subsystems/ins/ins_int.c

```diff
@@ -448,33 +471,45 @@ void ins_int_update_gps(struct GpsState *gps_s)
 void ins_int_update_gps(struct GpsState *gps_s __attribute__((unused))) {}
 #endif /* USE_GPS */
 
+/** agl_cb
+ * This callback handles all estimates of the height of the vehicle above the ground under it
+ * This is only used with the extended version of the vertical float filter
+ */
+#if USE_VFF_EXTENDED
+static void agl_cb(uint8_t sender_id, float distance) {
+  if (distance <= 0 || !(ins_int.baro_initialized)) {
+    return;
+  }
 
+if (sender_id == AGL_SONAR_ADC_ID ||
```

Comment:
Would be better to avoid the explicit list of sonar ID I think. Do you imagine the case where you have to two types of sonar/agl sensors at the same time ?

Reply:
This was done since the previous define was USE_SONAR and the defines to use the value depending on the altitude was also specific to sonar. I can replace the sonar defines with AGL defines but indeed, what if you have multiple sensors that can measure agl. We have had drones with sonar, laser and downward facing stereo camera, all which can provide agl.

Advice on how best to handle that config?